aten/src/ATen/templates/Tensor.h

```diff
@@ -75,6 +75,9 @@ struct Tensor : public detail::TensorBase {
   inline Tensor toType(ScalarType t) const;
   inline Tensor toBackend(Backend b) const;
 
+  /// Returns true if the `Tensor` is actually a `torch::autograd::Variable`.
+  bool isVariable() const noexcept;
```

Comment:
Let's stick with `snake_case`. I know some of the Tensor functions written manually above are camelCase, but the majority of functions on Tensors and Variables are snake_case.